test/base/integration_util.py

```diff
@@ -21,6 +22,16 @@ def skip_if_jenkins(cls):
     return cls
 
 
+def skip_unless_executable(executable):
+    if which(executable):
+        return lambda func: func
```

Comment:
This is a wired syntax. For everyone that is searching for it: https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures